src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -67,6 +67,14 @@
                 localStorage.setItem(profilerStorageKey + name, value);
             },
 
+            addEvent = function (element, eventName, callback) {
+                if (document.all) {
```

Comment:
@aik099 This is why I suggested checking on document rather than element (document is available at definition time). The benefit is that you choose the right implementation of the function only once instead of doing the check for each call